client_code/utils/_component_helpers.py

```diff
@@ -207,6 +208,10 @@ def walk(component_or_components):
 
 
 def _css_length(v):
+    if isinstance(v, str) and re.match(
+        r"\b\d+(\.\d+)?\s*(cm|mm|in|pt|pc|px|em|rem|vw|vh|vmin|vmax|%)\b", v
+    ):
+        return v
```

Comment:
We dont need this. The try will fail in the next step and will return the original value.

Reply:
Yeah, I misread that block, just reverted that part.